beam/tests/setup.py

```diff
@@ -570,3 +571,17 @@ def create_purchase_receipt_for_received_qty_test(settings):
 		},
 	)
 	pr.save()
+
+
+def create_network_printer_settings(settings):
+	printer_settings = [
+		{"name": "Primary Example Printer", "server_ip": "localhost", "port": 8888},

```

Comment:
Receiving Printer and Kitchen Printer

Reply:
@agritheory - updated in latest push (both versions are good)